src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -2328,6 +2328,18 @@ void Convert::pedalFromMEI(engraving::Pedal* pedal, const libmei::Pedal& meiPeda
         pedal->setEndHookType(engraving::HookType::HOOK_90);
     }
 
+    // @func
+    if (meiPedal.GetFunc() == "soft") {
+        if (pedal->beginText() != "") {
+            pedal->setBeginText(u"una corda");
+        }
+    } else if (meiPedal.GetFunc() == "sostenuto") {
```

Comment:
It seems you are not writing this back, are you?

Reply:
No, this is just for import. There is no specific style associated with it, and the terms used to indicate this are innumerable. I'll add a comment to clear this up.